internal/nhooyr.io/websocket/conn.go

```diff
@@ -141,6 +149,12 @@ func newConn(cfg connConfig) *Conn {
 		c.close(errors.New("connection garbage collected"))
 	})
 
```

Comment:
We used the options patterns set `inactivityTimeout`, in order to avoid more locks due to the `inactivityTimeout` race condition created in `c.timeoutLoop()`:

```
c.wg.Add(1)
go func() {
	defer c.wg.Done()
	c.timeoutLoop()
}()
```